pkgs/servers/gerbera/default.nix

```diff
@@ -0,0 +1,101 @@
+{ stdenv, lib, fetchFromGitHub
+, sqlite, expat
+, libuuid, libupnp, pugixml
+, cmake, pkgconfig
+, avcodecSupport ? false, libav ? null
+, curlSupport ? true, curl ? null
+, exifSupport ? true, libexif ? null
+, exiv2Support ? false, exiv2 ? null
+, ffmpegthumbnailerSupport ? true, ffmpegthumbnailer ? null
+, inotifySupport ? true, inotify-tools ? null
+, jsSupport ? true, duktape ? null
+, lastfmSupport ? false, liblastfm ? null
+, magicSupport ? true, file ? null
+, matroskaSupport ? true, libmatroska ? null, libebml ? null
+, mysqlSupport ? false, mysql ? null
+, systemdSupport ? false, systemd ? null
```

Comment:
Seems like an odd default for NixOS... but I know there are people who don't like setting that to `true` by default... :man_shrugging:

Reply:
This is the package not the NixOS module, the systemd dep here is as far as I can tell mysteriously useless but still an option.